PythonWrapperGenerator/src/EnvisionAstConsumer.cpp

```diff
@@ -125,15 +125,29 @@ void EnvisionAstConsumer::HandleClassDecl(clang::CXXRecordDecl* classDecl)
 						possibleAttributeSetters << methodName;
 				}
 			}
+			QMultiHash<QString, clang::CXXMethodDecl*> overloads;
 			// check if we have some more attributes which don't have an attribute macro:
 			for (auto method : classDecl->methods())
 			{
-				// Ignore private methods:
-				if (method->getAccess() == clang::AccessSpecifier::AS_private) continue;
+				// Ignore de-/constructors & deleted methods & overloaded methods:
```

Comment:
Here you say you ignore overloaded methods, but below you ignore overloaded operators. Which one is it?